user_management/src/user/views.py

```diff
@@ -22,6 +22,7 @@ def post(self, request):
             user = User.objects.create(username=json_request['username'],
                                        email=json_request['email'],
                                        password=json_request['password'])
+            user.save()
```

Comment:
User.objects.create() already adds instance to database.
Remove this line

Reply:
[dc77dca](https://github.com/tdameros/42-transcendence/pull/38/commits/dc77dca7ca1c6679f136801fb0909ed31c54a4a9)